packages/apps/comment-nuke/src/nuke.ts

```diff
@@ -1,13 +1,128 @@
-import { Comment, Devvit } from '@devvit/public-api';
-
+import { Comment, Devvit, Post } from '@devvit/public-api';
+ 
 export type NukeProps = {
   remove: boolean;
   lock: boolean;
   skipDistinguished: boolean;
   commentId: string;
   subredditId: string;
 };
-
+ 
+export type NukePostProps = {
+    remove: boolean;
+    lock: boolean;
```

Comment:
can a comment be both removed AND locked? I saw the conditional requiring at least one or the other but what if both are active?